tenable/io/scans.py

```diff
@@ -333,7 +334,7 @@ def create_scan_schedule(self, enabled=False, frequency=None, interval=None,
         return schedule
 
     def configure_scan_schedule(self, id, enabled=None, frequency=None, interval=None,
```

Comment:
can we give this parameter id as scan_id

Reply:
done